packages/client/bin/cli.ts

```diff
@@ -175,11 +175,14 @@ async function run() {
   }
 
   const common = new Common({ chain, hardfork: 'chainstart' })
+  const datadir = args.datadir ?? Config.DATADIR_DEFAULT
```

Comment:
Won't make this a blocker, but this shouldn't be necessary since `datadir` has a local CLI-specific default (first was afraid that this would actually *change*  the default, but this is not the case)? 🤔